web/html/src/branding/css/base/variables.scss

```diff
@@ -119,6 +119,11 @@ $input-height: 35px;
 $input-placeholder: $gray;
 :root {
   --input-selected: $eos-bc-green-500;
+
+  /* Bootstrap border variables */
```

Comment:
I think this is technically correct, but we need to be very careful here not to break existing styles. Historically we didn't do this because the old theme relied in too many different places on this not being the case. In short, if we can have this change, great, but we need to check that this doesn't break other things in other places because these are very global styles.